appium/sample-scripts/python/README.md

```diff
@@ -8,102 +10,183 @@ You'll find here all steps you need to start running your mobile tests
-from Testdroid cloud. Also you'll need to set the full path to your
-mobile app. This can be an Android or iOS application.
+The below example shows how to upload a hybrid Android demo app to Testdroid Cloud.
 
-```sh
-$ python upload.py 
-Filename to use in testdroid capabilities in your test: 719f52c4-2c34-4c25-b90b-08884f049d3a/Testdroid.apk
+```bash
+$ python upload.py -k xg8x...YXto -u ../../../apps/builds/Testdroid.apk
+Filename to use in testdroid capabilities in your test: 719f52c4-43c2-4c25-b91b-08884f049d3a/Testdroid.apk
 ```
 
-From the response message you need to store the application's file
-name in the cloud. In this example upload it is
-'719f52c4-2c34-4c25-b90b-08884f049d3a/Testdroid.apk'.
+The response message provides the application's cloud file
+name that is needed to start testing against the app. From the above example the path to store is:
+'719f52c4-43c2-4c25-b90b-08884f049d3a/Testdroid.apk'.
 
-## Common Settings
 
-There are some common settings that you need to set in all scripts
-regardless of the app type that you are testing. Each testdroid_*.py
-file needs to be updated with the following values.
+# Common Settings in Example Tests
 
-Here are all the values that you need to edit:
+There are some common settings needed to run any of the example tests, 
+regardless of the app type being tested. Each `testdroid_*.py`
+file needs to have the following values set as environment variables. Values can also be given as command line parameters to `run-test.py` script.
 
-* *screenshotDir* - where should screenshots be stored on your local drive
+Common values used in tests:
+
+* *screenshot_dir* - where should screenshots be stored on your local drive
 
 * *testdroid_username* - your email that you registered with to
    Testdroid Cloud.  **Rather use testdroid_apiKey.**
 
 * *testdroid_password* - your Testdroid Cloud password. **Rather use
    testdroid_apiKey.**
 
-* *testdroid_apiKey* - a personal unique key that allows you to
-   connect to Testdroid Cloud without the need to use your username
-   and passwords in your tests. You can find your api key under "My
-   account" in [Testdroid Cloud](https://cloud.testdroid.com/) UI.
-
-* *testdroid_project* - you can set this value as an environment variable
-  or set in the script. This is the name you want to give to your
-  project in Testdroid Cloud. Each project must have a unique name, 
-  which can also be modified later
+* *testdroid_apiKey* - a personal unique key allowing you to connect
+   to Testdroid Cloud without using username and passwords in
+   tests. Api key is found under "My account" in [Testdroid
+   Cloud](https://cloud.testdroid.com/) UI.
+
+* *testdroid_project* - the project name in Testdroid Cloud. Each
+  project must have a unique name, which can be modified (in Cloud)
+
+* *testdroid_testrun* - name of this test run inside of
+  `testdroid_project`. Each test run can have it's own name (eg. date + time)
```

Comment:
Shall we instead say: "Each test run can have the same name, but it is recommended to set it dynamically (e.g. with timestamp or device name)